src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -55,6 +60,8 @@ public void createPrimaryUserAffiliation(String eventPayload) {
         return;
       } else {
         userTenantService.createPrimaryUserTenantAffiliation(consortiaTenant.getConsortiumId(), consortiaTenant, userEvent);
+        var permissionUser = PermissionUser.of(UUID.randomUUID().toString(), userEvent.getUserDto().getId(), List.of(PERMISSION_NAME));
+        permissionsClient.create(permissionUser);
```

Comment:
1. I would suggest to update in case permissions exist or create if not
2. does it make sense to move this logic to createPrimaryUserTenantAffiliation?

Reply:
as discussed this permission creation is not required anymore